x-pack/plugins/fleet/public/applications/fleet/sections/settings/hooks/index.tsx

```diff
@@ -1,6 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
```

Comment:
is this a mistake to remove the copyright?

Reply:
This file is empty so I'm just removing it entirely 😃